recipes/repository.rb

```diff
@@ -49,8 +49,15 @@
 when 'redhat', 'centos'
   # CentOS 6 and 7 have Gluster in the Storage SIG instead of a gluster hosted repo
   if node['platform_version'].to_i > 5
-    repo_url = "http://mirror.centos.org/centos/$releasever/storage/$basearch/gluster-#{node['gluster']['version']}/"
-    gpg_url = 'https://raw.githubusercontent.com/CentOS-Storage-SIG/centos-release-storage-common/master/RPM-GPG-KEY-CentOS-SIG-Storage'
+    if Chef::VersionConstraint.new('>= 3.9').include?(node['gluster']['version'])
+      subdomain = 'buildlogs'
```

Comment:
`buildlogs` is included as the "Testing" repository in the centos-release-gluster package and given that there is no such package for 3.9 on the mirrors yet, I figured it was still in testing packaging-wise. Here is [the repository configuration](https://github.com/CentOS-Storage-SIG/centos-release-gluster39/blob/master/CentOS-Gluster-3.9.repo).